scripts/app.js

```diff
@@ -167,6 +167,7 @@ function retreiveBlogs(){
                 console.log(profile.data().Names)
```

Comment:
Please do not log to console, except in development

Reply:
Eeehh ok! I didn't know. I will kindly correct it